src/test/java/teammates/ui/webapi/QueryLogsActionTest.java

```diff
@@ -155,11 +241,131 @@ protected void testExecute() throws Exception {
         assertEquals(errorLogSourceLocation2, entry6.getSourceLocation());
         assertEquals(errorLogTimestamp2, entry6.getTimestamp());
 
+        // Test hiding of sensitive fields for non-admin maintainer
```

Comment:
No need for this comment. `______TS` already explained it.

Reply:
Ok. It is updated.